test/io/fixtures.sql

```diff
@@ -198,3 +198,9 @@ $$ language sql set statement_timeout = '4s';
 create function get_postgres_version() returns int as $$
   select current_setting('server_version_num')::int;
 $$ language sql;
+
+GRANT SET ON PARAMETER log_min_duration_sample TO postgrest_test_anonymous;
+
+create or replace function log_min_duration_test() returns text as $$
+  select current_setting('log_min_duration_sample',false);
+$$ language sql set log_min_duration_sample = '5s';
```

Comment:
Since this causes an error on older pg versions, maybe you can use [work_mem](https://www.postgresql.org/docs/current/runtime-config-resource.html#GUC-WORK-MEM) instead.

IIRC it won't require a GRANT.

```suggestion
create or replace function work_mem_test() returns text as $$
  select current_setting('work_mem',false);
$$ language sql set work_mem = '6000';
```